content/docs/actors/development/input_schema.md

```diff
@@ -166,6 +166,7 @@ Properties:
 | `enum`       | [String]                                                                                  | Required if <br/>`editor` <br/>is `select` | Using this field, you can limit values <br/>to the given array of strings. <br/>Input will be displayed as select box.   |
 | `enumTitles` | [String]                                                                                  | No                                         | Titles for the `enum` keys described.                                                                                    |
 | `nullable`   | Boolean                                                                                   | No                                         | Specifies whether `null` <br/>is an allowed value.                                                                       |
+| `isSecret`   | Boolean                                                                                   | No                                         | Specifies whether the input field will be stored encrypted. Only available with `textfield` and `textarea` editors.      |
```

Comment:
Please add some newlines here like in the other entries to avoid horizontal scroll in the table